widgetastic_manageiq/__init__.py

```diff
@@ -61,6 +61,58 @@
 from cfme.exceptions import ItemNotFound
 
 
+class SnapshotMemorySwitch(Widget):
+    """ This replaces the old-style checkbox when working with snapshot memory.
+
+    At first glance on the page it looks like a BootstrapSwitch but it really isn't.
+    Basic switch controls are implemented here along with fill and read methods,
+    so this should be usable as any other widget.
+    """
+
+    LOCATOR = u'//div[contains(@class, "bootstrap-switch-snap_memory")]'
+    ON_LOCATOR = u'//div[contains(@class, "bootstrap-switch-on")]'
+
+    def __init__(self, parent, logger=None):
```

Comment:
Unnecessary constructor since there are no statements made. If this constructor were to be included, it should make the call to the parent class `__init__` through `super()`, not directly through the class name.